### ISSUE TITLE
sync support in yowsup-2 branche is broken

### ISSUE THREAD

**Issue by NeoBelerophon** [stv0g/transwhat#25]
the sync method in buddylist is missing:
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/opt/transwhat/bot.py", line 78, in _sync
    count = self.session.buddies.sync(user, password)
AttributeError: 'BuddyList' object has no attribute 'sync'
```

**Comment by moyamo** [comment]
Is this when you call \sync? Transwhat syncs your contacts automatically. So you never need to call sync manually. I think I'll just remove the sync command.

**Comment by NeoBelerophon** [comment]
Yes. If it is not longer used removing this command would be best solution

**Comment by moyamo** [comment]
Fixed 8e9e8a6476bbd76d4433a564ca306e7112674c96
